### PR TITLE
L2D24A3

### PR DESCRIPTION
**Pull request by evyryan**
Fixed fatheads
added question headers
one paragraph needs to be added
4.2.24 - Fixed edit that I shouldn't have accepted

### CODE REVIEW

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 1]
```diff
@@ -98,8 +99,8 @@
       <lb ed="#V" n="25"/>actio ante intelligere. 
    </p>
                <p xml:id="kzz7yh-f70880-d1e170">
-                  <lb ed="#V" n="26"/>Contra prius ordine nature est conuersio 
-      intelleciu<lb ed="#V" n="27" break="no"/>ad intelligibile: quam ipsum intelligat. Sicut 
+                  <lb ed="#V" n="26"/>Contra prius ordine nature est conuersio itellecius <!--should have t?-->
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:102]:
```suggestion
                  <lb ed="#V" n="26"/>Contra prius ordine nature est conuersio <sic>intellecius</sic>

Comment (RobbieBuhite) [reply kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:102]: Yeah, I think it is supposed to be "intellectius," but there is clearly no "t."

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 2]
```diff
@@ -204,10 +206,10 @@
...
    </p>
                <p xml:id="kzz7yh-f70880-d1e375">
-                  <lb ed="#V" n="108"/>Qui vult tenere primam opinionem potest dice 
-      <lb ed="#V" n="109"/>re ad primum ad partem secundam quod 
-      <lb ed="#V" n="110"/>preter illam dispositionem que requiritur in potentia ad e 
-      <lb ed="#V" n="111"/>ceptionem speciet non est alia conuersio: nisi apprehensio: et 
+                  <lb ed="#V" n="108"/>Qui vult tenere primam opinionem potest  
+      dice<lb ed="#V" n="109" break="no"/>re ad primum ad partem secundam quod 
+      <lb ed="#V" n="110"/>preter illam dispositionem que requiritur in potentia ad 
+      e<lb ed="#V" n="111" break="no"/>ceptionem speciet non est alia conuersio: nisi apprehensio: et
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:212]:
```suggestion
      <sic>e<lb ed="#V" n="111" break="no"/>ceptionem</sic> speciet non est alia conuersio: nisi apprehensio: et 
```

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 3]
```diff
@@ -230,9 +232,10 @@
             </div>
             <div xml:id="kzz7yh-f70880-Dd1e423">
                <head xml:id="kzz7yh-f70880-Hd1e425">Quaestio 2</head>
+               <head type="question-title">Utrum nostrum intelligere sit agere</head>
                <p xml:id="kzz7yh-f70880-d1e427">
       ¶ Quaestio II. 
-      <lb ed="#V" n="127"/>Ecundo queritur. vtrum 
+      <lb ed="#V" n="127"/>SEcundo queritur vtrum 
       no<lb ed="#V" n="128" break="no"/>strum 
       in<lb ed="#V" n="129" break="no"/>telligere sit agere. Et videtur quod sic: secundum philosophum
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:240]:
```suggestion
      in<lb ed="#V" n="129" break="no"/>telligere sit agere. Et videtur quod sic: secundum <ref>philosophum 
```

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 4]
```diff
@@ -230,9 +232,10 @@
...
             <div xml:id="kzz7yh-f70880-Dd1e423">
                <head xml:id="kzz7yh-f70880-Hd1e425">Quaestio 2</head>
+               <head type="question-title">Utrum nostrum intelligere sit agere</head>
                <p xml:id="kzz7yh-f70880-d1e427">
       ¶ Quaestio II. 
-      <lb ed="#V" n="127"/>Ecundo queritur. vtrum 
+      <lb ed="#V" n="127"/>SEcundo queritur vtrum 
       no<lb ed="#V" n="128" break="no"/>strum 
       in<lb ed="#V" n="129" break="no"/>telligere sit agere. Et videtur quod sic: secundum philosophum 
       <lb ed="#V" n="130"/>primo ethicorum ca. 9. felicitas consistit in actu.
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:241]:
```suggestion
      <lb ed="#V" n="130"/>primo ethicorum ca. 9.</sic> felicitas consistit in actu. 
```

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 5]
```diff
@@ -457,7 +460,7 @@
                <p xml:id="kzz7yh-f70880-d1e838">
                   <lb ed="#V" n="5"/>Ad primum dicendum quod intellectus possibilis 
       <lb ed="#V" n="6"/>maxime reductus in actum per 
-      <lb ed="#V" n="7"/>similitudinem: non tantummodo est passiuus: sed etiam 
+      <lb ed="#V" n="7"/>similitudinem: non tantum modo est passiuus: sed etiam
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:463]:
```suggestion
      <lb ed="#V" n="7"/>similitudinem: non tantummodo est passiuus: sed etiam 
```

Comment (RobbieBuhite) [reply kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:463]: These can be separate words, but often they get combined when they're next to each other.

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 6]
```diff
@@ -963,6 +968,7 @@
             </div>
             <div xml:id="kzz7yh-f70880-Dd1e1766">
                <head xml:id="kzz7yh-f70880-Hd1e1768">Quaestio 5</head>
+               <head type="question-title">Utrum intellectus noster argumetando discurrens a creaturis ad creatorem</head>
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:971]:
```suggestion
               <head type="question-title">Utrum intellectus noster argumentando discurrens a creaturis ad creatorem in fine discursus intelligat per suam naturalem virtutem diuinam essentiam immediate</head>
```

File: kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml [hunk 7]
```diff
@@ -1016,7 +1022,7 @@
       <lb ed="#V" n="114"/>sus non intelligimus immediate essentiam creatoris. 
    </p>
                <p xml:id="kzz7yh-f70880-d1e1861">
-                  <lb ed="#V" n="115"/>Respondeo quod quamuis homo bene vtendo 
+                  <lb ed="#V" n="115"/>Respondeo quod quamuis homo bene vtendo <!--word-->
```
Comment (RobbieBuhite) [inline kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:1025]:
```suggestion
                  <lb ed="#V" n="115"/>Respondeo quod quamuis homo bene vtendo
```

Comment (RobbieBuhite) [reply kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml:1025]: This is the gerund of the deponent verb, "utor."

**Review by RobbieBuhite — CHANGES_REQUESTED** [review]

**Review by jeffreycwitt — APPROVED** [review]

**Pull request merged**